src/main/kotlin/traffic_simulation/RoadNetwork.kt

```diff
@@ -2,33 +2,50 @@ package traffic_simulation
 
 class RoadNetwork(val capacity: Int) {
 
-    fun calculateDemand(vehicles: List<Vehicle>): Int {
-        var capacityDemand: Int = 0
+    fun calculateDemandFor24Hours(vehicles: List<Vehicle>): MutableMap<Int , Int> {
```

Comment:
This solves a different issue and I don't think it does that very well. Could you revert that file back to what it was before your pull request and try to make the smallest possible changes that let the code compile (even if the model doesn't do anything meaningful yet).

(Argh, babylonisches Sprachwirrwarr... You get my point.)